.travis.yml

```diff
@@ -24,8 +24,12 @@ install:
   - docker exec -t host bash -c "yes | apt-get update"
   - docker exec -t host bash -c "yes | apt-get upgrade"
   - docker exec -t host bash -c "yes | apt-get install git clang-format-6.0"
+  - sudo curl -L -o /tmp/wake.deb https://github.com/sifive/wake/releases/download/v0.19.0/ubuntu-16-04-wake_0.19.0-1_amd64.deb
+  - sudo apt install /tmp/wake.deb
 
 # Here's where we actually run the test.
 script:
 # Check source code formatting
   - docker exec -t host bash -c "cd /travis && ./scripts/check-format"
+# Run dummy Wake program in order to run Wake type cehcker.
```

Comment:
minor spelling "cehcker"

Reply:
Ah, good catch, sorry about that!

```suggestion
# Run dummy Wake program in order to run Wake type checker.
```

Reply:
This change only adds CI for the Wake rule using Wake 0.19, so it doesn't have any effect on the actual Wake code itself. Wake 0.19 is almost completely compatible with Wake 0.18, and therefore Wake 0.17.2. There is some older, Wake 0.17 syntax that is no longer valid, but Wake 0.17.2 is an intermediate point at which you can write plenty of code that is compatible across 0.17–0.19.